src/commands/lint/configureLintingToolsCommand.ts

```diff
@@ -0,0 +1,194 @@
+                        `Updated developer dependency in package.json. Run package manager such as npmr/yarn/pnpm to update node modules.`,
+                        MessageType.InformationOk
+                    );
+                } else {
+                    await this.showMessage(
+                        `No update was made in package.json. It already includes eslint plugin for LWC data graph analysis.`,
+                        MessageType.InformationOk
+                    );
+                }
+
+                return Promise.resolve(true);
+            }
+        } catch (error) {
+            await this.showMessage(
+                `There was an error trying to update developer dependency in package.json: ${error}`
+            );
+            return Promise.resolve(false);
+        }
+    }
+
+    static updateDevDependencies(): boolean {
+        const packageJson: PackageJson = WorkspaceUtils.getPackageJson();
+        const devDependencies = packageJson.devDependencies;
+        let modified = false;
+
+        if (devDependencies) {
+            eslintDependencies.forEach((nameValuePair) => {
+                const [name, value] = nameValuePair;
+                if (!devDependencies[name]) {
+                    devDependencies[name] = value;
+                    modified = true;
+                }
+            });
+        }
+
+        if (modified) {
+            // Save json only if the content was modified.
+            WorkspaceUtils.setPackageJson(packageJson);
+        }
+
+        return modified;
+    }
+
+    static updateEslintrc() {
+        const eslintrcPath = path.join(
+            WorkspaceUtils.getWorkspaceDir(),
+            WorkspaceUtils.LWC_PATH,
+            '.eslintrc.json'
+        );
+
+        if (fs.existsSync(eslintrcPath)) {
+            const eslintrc = JSON.parse(fs.readFileSync(eslintrcPath, 'utf-8'));
+            const eslintrcExtends = eslintrc.extends as Array<string>;
+
+            if (eslintrc && eslintrcExtends) {
+                let modified = false;
+
+                if (!eslintrcExtends.includes(eslintRecommended)) {
+                    eslintrcExtends.push(eslintRecommended);
+                    modified = true;
+                }
+
+                if (!eslintrcExtends.includes(lwcGraphAnalyzerRecommended)) {
+                    eslintrc.extends.push(lwcGraphAnalyzerRecommended);
+                    modified = true;
+                }
+
+                if (modified) {
+                    // Save json only if the content was modified.
+                    fs.writeFileSync(
+                        eslintrcPath,
+                        JSON.stringify(eslintrc, null, TAB_SPACES)
+                    );
+                }
+            }
+        } else {
+            // Create eslintrc
+            fs.writeFileSync(
+                eslintrcPath,
+                `{"extends": ["${eslintRecommended}", "${lwcGraphAnalyzerRecommended}"]}`
```

Comment:
To be consistent, it feels like we should build this up and save in the same format as we do for existing files. I.e. make a JSON object, and stringify it with `TAB_SPACES` to write it to disk.